src/vue/backend/src/router/index.js

```diff
@@ -91,8 +92,8 @@ const routes = [
                 children: [
                     {
                         path: 'api',
-                        name: 'Connect Account'
-                        // component
+                        name: 'Connect Account',
+                        component: GuidedSetupConnectApi
```

Comment:
This currently does not render on `/guided-setup/connect/api`, Lee is making a refactoring ticket for these routes. Wait for that and verify that your view is rendering at the expected link